app/helpers/application_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+  # Returns the full title on a per-page basis.
+  def full_title page_title = ""
+    base_title = I18n.t("main_title")
+    if page_title.empty?
```

Comment:
code ngắn thì dùng toán tử 3 ngôi để viết trên 1 line cho gọn nhé
page_title.empty? ? base_title : "#{page_title} | #{base_title}"